src/main/scala/ChatBot.scala

```diff
@@ -0,0 +1,32 @@
+object ChatBot extends App {
+
+  val botName = "Matilda"
+  val randomAns = Array("I don’t know, let me google it up for you","I don’t know and I’m too lazy to look it up...")
+  var yell = 0
+var quit = true
+
+  println(s"Hi I am $botName, I am here to help. Ask me anything!")
+
+  def pickAns(): Int = {
+    val r = new scala.util.Random
+    val r1 = r.nextInt(2)
+    r1
+  }
+
+    while(quit) {
+      var question = readLine()
```

Comment:
this could be a val as in every iteration of while loop you will readLine() and you aren't reassigning the value anywhere to it therefore this shouldn't be a val